packages/server/src/db/db.service.ts

```diff
@@ -21,13 +27,21 @@ export default class DBService {
   private passwordResetRepository: Repository<PasswordResetEntity>;
   private socialProfileRepository: Repository<SocialProfileEntity>;
   private emailResetRepository: Repository<EmailResetEntity>;
+  private shelfRepository: Repository<ShelfEntity>;
+  private flowerRepository: Repository<FlowerEntity>;
+  private shelfInvitationRepository: Repository<ShelfInvitationEntity>;
+  private userToShelfRepository: Repository<UserToShelfEntity>;
 
   constructor() {
     this.userRepository = getRepository(UserEntity);
     this.emailResetRepository = getRepository(EmailResetEntity);
     this.emailVerificationRepository = getRepository(EmailVerificationEntity);
     this.passwordResetRepository = getRepository(PasswordResetEntity);
     this.socialProfileRepository = getRepository(SocialProfileEntity);
+    this.shelfRepository = getRepository(ShelfEntity);
+    this.flowerRepository = getRepository(FlowerEntity);
+    this.shelfInvitationRepository = getRepository(ShelfInvitationEntity);
+    this.userToShelfRepository = getRepository(UserToShelfEntity);
```

Comment:
I would suggest splitting this service, in scope of future PR's.
src/db/auth.service
src/db/user-management.service
src/db/shelf.service
src/db/flower.service
Or create some kind of CRUD DB service.

Reply:
Agree, it asks for refactoring, sepecially the test part.